### DISCUSSION TITLE
integration of presence detectors

### DISCUSSION THREAD

**Discussion by KStaudenmaier** [JoDehli/PyLoxone#311] [Ideas]
Hi, 

first of all, thank you for the opportunity to integrate loxone into home assistant.

Would it be possible to integrate presence detectors into pyloxone so that, for example, lighting control could also be automated here?

Thanks.

Kind regards
SkyFox

**Comment by JoDehli** [comment]
@KStaudenmaier please look for custom sensor on the main page and search in already closed issues.
